newspaper/models.py

```diff
@@ -380,19 +376,34 @@ class ArticlePage(RoutablePageMixin, Page):
         index.SearchField('authors'),
     ]
 
-    @route(r'^(\d{4})/(\d{2})/(\d{2})/(.+)/$')
-    def dated_article_with_slug(self, request, year, month, day, slug):
-        pprint(vars(self))
-        post_page = self.get_posts().filter(slug=slug).first()
-        if not post_page:
-            raise Http404
-        return Page.serve(post_page, request, *args, **kwargs)
 
-    def post_by_date_slug(self, request, year, month, day, slug, *args, **kwargs):
-        post_page = self.get_posts().filter(slug=slug).first()
-        if not post_page:
-            raise Http404
-        return Page.serve(post_page, request, *args, **kwargs)
+class ArticleAuthorRelationship(models.Model):
+    """
+
+    Intermediate table supporting ManyToMany between Articles and Authors
```

Comment:
Remind me again, what was the issue that necessitated this class? Just curious :)

Reply:
I couldn't get Wagtail to nest the authors of an article in the API response when the relation was a ManyToManyField :/

It's potentially possible, but wagtail had a few docs regarding nesting ForeignKey relationships in the API, so I made an intermediary model with two ForeignKey relationships (well, one of them is ParentelKey which is just a wagtail extension of ForeignKey with support for inlining in the CMS)